### PR TITLE
if a default value is provided, don't expand the dropdown on page load

### PR DESCRIPTION
**Pull request by zdhickman**
Following up on https://github.com/fmoo/react-typeahead/issues/151, this is an implementing patch if you'd like it!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/fmoo/react-typeahead.git"
+    "url": "https://github.com/HubSpot/react-typeahead.git"
```
Comment (fmoo) [inline package.json]: nope

**Comment by fmoo** [comment]
Please leave the lib/ out of the PR; lib is compiled as part of `prepublish` automatically.

@nosilleg - Was thinking of getting this into the 1.1.x release; any concerns if I do?

**Comment by nosilleg** [comment]
What is the usage scenario for this change?  This seems like it would impact a lot of different use cases.  Is there an issue (ticket) for this?  If not, can an issue be created so that we can discuss the validity of the functionality change?  (as opposed to addressing the code in here)

@fmoo Thanks for including me on this.  I've got other changes that are part completed for the next rev of the component (obvs pending your approval) one of which is the potential removal of defaultValue if/when I confirm that it's redundant in the controlled component version.

**Comment by fmoo** [comment]
@nosilleg - I think the main issue is #151 (also #21?), but there are 2-3 other reports/questions around typeahead on textentry.blur / non-focus that could be related (#169, #154, #129, #115, #113, and maybe #104)

File: package.json (hunk 1 above)
Comment (zdhickman) [inline package.json]: Ah, the last two commits in this PR should not be here. I'll clean that up.

**Pull request closed without merging**